.gitignore

```diff
@@ -309,15 +309,7 @@ src/AltinnRepositories/gitea-data/gitea/*
 src/Altinn.Platform/Altinn.Platform.Receipt/Receipt/wwwroot/receipt/js/react
 src/Altinn.Platform/Altinn.Platform.Receipt/Receipt/wwwroot/receipt/css
 
-######### Built react apps for designer ###########
-src/studio/src/designer/backend/wwwroot/designer/css/react
-src/studio/src/designer/backend/wwwroot/designer/js/react
-src/studio/src/designer/backend/wwwroot/designer/js/lib
-src/studio/src/designer/backend/wwwroot/designer/css/lib
-src/studio/src/designer/backend/wwwroot/designer/css/font-awesome
-src/studio/src/designer/backend/wwwroot/designer/css/fonts
-src/studio/src/designer/backend/wwwroot/designer/css/bootstrap*.css
-src/studio/src/designer/backend/wwwroot/designer/frontend
+
```

Comment:
I'd just keep these lines in case someone has generated files in these locations and has an habit of using `git add .`

The files should be deleted, but keeping them does not create harm, so no reason to force everyone to clean it.

Reply:
Good point. 👍